charts/rollups-node/Chart.yaml

```diff
@@ -13,8 +13,7 @@ maintainers:
     url: https://github.com/oap75
 
 type: application
-version: 0.5.2
-appVersion: "0.9.1"
+version: 1.0.0
```

Comment:
what is the version strategy?
version is now 1.0.0 but will follow its own number independent of rollups-node?
Should we have an `appVersion: "1.0.0"`?

Reply:
The idea is to sync with cartesi/rollups-node, in case we have new releases, we increment a `-N` in the end, like

`1.0.0-1`, `1.0.0-2`, ....

Reply:
Not my best preference, but ok.